docs/manual/operations.md

```diff
@@ -189,6 +189,15 @@ This will make sure that the processes are proactively excluded, instead of wait
 
 _NOTE_: You should always set the processes under maintenance before setting the maintenance mode. See [Internals](#internals) for more details.
 
+## Delaying the shutdown of the Pod
+
+When using the [unified image](./customization.md#unified-vs-split-images) the `fdb-kubernetes-monitor` supports to delay the shutdown of itself.
+The idea is to shutdown the fdbserver processes immediately, but keep the `fdb-kubernetes-monitor` process running for the specified `foundationdb.org/delay-shutdown`.
```

Comment:
I'm a bit unclear here, this makes it sound like it will just call for the fdbserver processes to get immediately shut down in any case, but below it makes it sound like changing the delay for the monitor somehow helps it with failed fdbserver processes, and I don't understand the benefit of detecting before shutting down

Reply:
When the operator is doing an update on all Pods, it performs different checks e.g. https://github.com/FoundationDB/fdb-kubernetes-operator/blob/main/controllers/update_pods.go#L193-L195 it checks if there are any Pods that are recently deleted, if so it will not update other Pods. In addition the operator checks the fault tolerance and will only update Pods if the fault tolerance is at the desired level.

When the operator is now updating Pods and it deletes a Pod to update it, there can a window where the Pod is already gone and the operator has not yet created the new Pod. In addition if the shutdown of the fdbserver processes is less than 60 seconds ago, the fdb cluster will report full fault tolerance (as 60 seconds is the default failure detection time in fdb). So if the operator runs the update pod subreconciler again it might take another set of pods down, as there is no pod with a deletion timestamp and the fault tolerance is up. By adding the delay shutdown, the pod will be "stuck" in deletion and the fdbserver process will be shutdown. This will reduce the risk of the race condition above.

Reply:
that makes total sense, I think we should add some reference to this race being a motivator for using the delayed shutdown (I don't know if there is other documentation or not, but any sort of allusion would go a long way I think(